library/swift/src/stats/Counter.swift

```diff
@@ -4,12 +4,14 @@ import Foundation
 @objc
 public protocol Counter: AnyObject {
   /// Increment the counter by the given count.
-  func increment(count: Int)
+  /// - returns: A status indicating if the action was successful.
+  func increment(count: Int) -> Int32
```

Comment:
At this layer I'd have it return void. We don't currently expose Envoy return codes beyond the bridge layer elsewhere, and they're only return codes there because of a lack of a platform-agnostic way to signal success, if relevant.